dynaconf/default_settings.py

```diff
@@ -137,6 +137,10 @@ def reload(*args, **kwargs):
     "db": int(get("REDIS_DB_FOR_DYNACONF", 0)),
     "decode_responses": get("REDIS_DECODE_FOR_DYNACONF", True),
 }
+redis_password = get("REDIS_AUTH_FOR_DYNACONF", None)
+if redis_password:
```

Comment:
```suggestion
if redis_password is not None:
```

so we allow empty strings as password.